Mathlib/Analysis/NormedSpace/Dual.lean

```diff
@@ -53,35 +53,15 @@ variable (E : Type*) [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
 variable (F : Type*) [NormedAddCommGroup F] [NormedSpace 𝕜 F]
 
 /-- The topological dual of a seminormed space `E`. -/
-def Dual :=
+abbrev Dual :=
   E →L[𝕜] 𝕜
 #align normed_space.dual NormedSpace.Dual
 
--- Porting note: added manually
-section DerivedInstances
+-- Note: helper below
+instance : NormedSpace 𝕜 (Dual 𝕜 E) := inferInstance
```

Comment:
Does something below timeout if this is absent? If so, can you make it clear that this "helper" is to resolve a performance problem?

Reply:
It fails on line 77:
```
failed to synthesize instance
  Norm (Dual 𝕜 E →L[𝕜] Dual 𝕜 E)
```

Reply:
With a failure or a timeout? (I can't open lean right now)

Reply:
It failed quickly. I didn't check further.

Reply:
That line can be fixed with:
```lean
theorem inclusionInDoubleDual_norm_eq :
    (by exact ‖inclusionInDoubleDual 𝕜 E‖) = ‖ContinuousLinearMap.id 𝕜 (Dual 𝕜 E)‖ :=
  ContinuousLinearMap.op_norm_flip _
```
This feels like a bug to me.

Reply:
The next failure in the file is even worse:
```
--ok
#check ContinuousLinearMap.norm_id_le (𝕜 := 𝕜) (E := Dual 𝕜 E)

theorem inclusionInDoubleDual_norm_le : ‖inclusionInDoubleDual 𝕜 E‖ ≤ 1 := by
  rw [inclusionInDoubleDual_norm_eq]
  -- fails
  have := ContinuousLinearMap.norm_id_le (𝕜 := 𝕜) (E := Dual 𝕜 E)
  exact this
```

Reply:
Referenced leanprover/lean4#2522 in the file